system-tests/src/utils/imageMetadataTools.ts

```
@@ -47,7 +47,12 @@ export async function imageSavedPageYCoordinate(pathToImage: string): Promise<nu
     }
     const coordinate = coordString.split("moocfi-page-y-")[1]
     if (coordinate) {
-      return Number(coordinate)
+      const yCoordinate = Number(coordinate)
+      if (isNaN(yCoordinate)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)